core/src/Streamly/Data/Unbox.hs

```diff
@@ -0,0 +1,27 @@
+-- |
+-- Module      : Streamly.Data.Unbox
+-- Copyright   : (c) 2022 Composewell Technologies
+--
+-- License     : BSD3
+-- Maintainer  : streamly@composewell.com
+-- Stability   : released
+-- Portability : GHC
+--
+-- This module provides the 'Unbox' type class for unboxing values in
+-- 'MutableByteArray'.
```

Comment:
for converting Haskell values to unboxed representation in a MutableByteArray and boxing values from MutableByteArray to create Haskell representation.